jlm/llvm/ir/operators/load.cpp

```diff
@@ -197,30 +197,66 @@ is_multiple_origin_reducible(const std::vector<rvsdg::output *> & operands)
   return states.size() != operands.size() - 1;
 }
 
+// s2 = store_op a v1 s1
+// v2 s3 = load_op a s2
+// ... = any_op v2
+// =>
```

Comment:
This comment kind of makes it appear as if the store disappears, I assume it doesn't?

Reply:
No, the store does not disappear. The only node that is removed is the load. I clarified in the function documentation by adding the store.